package.json

```diff
@@ -1,83 +1,91 @@
 {
   "name": "compression-webpack-plugin",
-  "version": "1.1.12",
-  "author": "Tobias Koppers @sokra",
+  "version": "2.0.0",
   "description": "Prepare compressed versions of assets to serve them with Content-Encoding",
-  "main": "dist/cjs.js",
-  "files": [
-    "dist"
-  ],
   "license": "MIT",
+  "repository": "webpack-contrib/compression-webpack-plugin",
+  "author": "Tobias Koppers @sokra",
+  "homepage": "https://github.com/webpack-contrib/compression-webpack-plugin",
+  "bugs": "https://github.com/webpack-contrib/compression-webpack-plugin/issues",
+  "bin": "",
+  "main": "dist/cjs.js",
+  "engines": {
+    "node": ">= 6.9.0 <7.0.0 || >= 8.9.0"
```

Comment:
We should relax this to just the minium major version of the `node` we support, respectively `"node": ">= 6"`

Reply:
Yeah and `node` v6.0.0 is currently the minimum supported LTS (Maintenance) according to the [node LTS schedule](https://github.com/nodejs/Release#release-schedule), v7.0.0 is EOL and v8.0.0 is the Active LTS. Why do we need to write a doctor thesis `">= 6.9.0 < 7.0.0 || >= 8.9.0"` here *by default* ?

Reply:
@michael-ciniawsky only `6.9` (Boron) is lts https://nodejs.org/en/download/releases/

Reply:
Yeah I know but the minor and patch versions are likely not relevant (by default) and will only result in issues opened why we are using e.g `6.9` in particular here and if e.g `6.8` doesn't work and why. If `6.8` does work (which is likely) the next issue opened will be that the package manger is displaying warnings about `engines` mismatching. If for a particular reason e.g a bug in `zlib` does require a specific minor version then we could always change it and justify our decision

Reply:
@michael-ciniawsky Because we already used this approach and it was not effective. We had to constantly find references to problems in nodejs bug tracker and increase minimum required nodejs version. Then we received issue about why one plugin/loader requires `nodejs@6.1` and another `nodejs@6.3`. Also, we constantly received reports about perf regression and weird about crash on non `lts` version. 

At the moment, we are losing time on this discussion since we have already discussed this several times. Also it is not place for these discussion (please open issue in `webpack-default` If you still want to continue the discussion).